.github/workflows/build.yaml

```diff
@@ -34,17 +34,26 @@ jobs:
       - name: Run unittests
```

Comment:
So unlike in the jenkins pipeline, in here we are running unit tests first and then the behave tests?

Reply:
I'm not sure of the order to be honest. But I don't think it matters too  much.